extensions/database/tests/src/com/google/refine/extension/database/DatabaseImportControllerTest.java

```diff
@@ -220,27 +219,25 @@ public void testDoPostCreateProject() throws IOException, ServletException {
     }
 
     @BeforeTest
-    @Parameters({ "mySqlDbName", "mySqlDbHost", "mySqlDbPort", "mySqlDbUser", "mySqlDbPassword", "mySqlTestTable" })
+    @Parameters({ "sqliteDbName", "sqliteDbHost", "sqliteDbPort", "sqliteDbUser", "sqliteDbPassword", "sqliteTestTable" })
     public void beforeTest(
-            @Optional(DEFAULT_MYSQL_DB_NAME) String mySqlDbName, @Optional(DEFAULT_MYSQL_HOST) String mySqlDbHost,
-            @Optional(DEFAULT_MYSQL_PORT) String mySqlDbPort, @Optional(DEFAULT_MYSQL_USER) String mySqlDbUser,
-            @Optional(DEFAULT_MYSQL_PASSWORD) String mySqlDbPassword, @Optional(DEFAULT_TEST_TABLE) String mySqlTestTable) {
+            @Optional(DEFAULT_MYSQL_DB_NAME) String sqliteDbName, @Optional(DEFAULT_MYSQL_HOST) String sqliteDbHost,
+            @Optional(DEFAULT_MYSQL_PORT) String sqliteDbPort, @Optional(DEFAULT_MYSQL_USER) String sqliteDbUser,
+            @Optional(DEFAULT_MYSQL_PASSWORD) String sqliteDbPassword, @Optional(DEFAULT_TEST_TABLE) String sqliteTestTable) {
 
         MockitoAnnotations.initMocks(this);
 
         testDbConfig = new DatabaseConfiguration();
-        testDbConfig.setDatabaseHost(mySqlDbHost);
-        testDbConfig.setDatabaseName(mySqlDbName);
-        testDbConfig.setDatabasePassword(mySqlDbPassword);
-        testDbConfig.setDatabasePort(Integer.parseInt(mySqlDbPort));
-        testDbConfig.setDatabaseType(MySQLDatabaseService.DB_NAME);
-        testDbConfig.setDatabaseUser(mySqlDbUser);
+        testDbConfig.setDatabaseHost(sqliteDbHost);
+        testDbConfig.setDatabaseName(sqliteDbName);
+        testDbConfig.setDatabasePassword(sqliteDbPassword);
+        testDbConfig.setDatabasePort(0); // unused for SQLite
```

Comment:
Would it then be worth removing the corresponding parameter from the method arguments, since it isn't used?